gradle/deployment.gradle

```diff
@@ -23,79 +23,58 @@ signing
     sign configurations.archives
 }
 
-uploadArchives
+publishing
```

Comment:
So I remember this upload step being very finicky, but that was years back. Have you had any chance to test this new approach?

Reply:
Short answer: no.

Longer answer: the comments indicated that it was due to two problems, https://github.com/travis-ci/travis-ci/issues/9555 and an issue with uploads to the Sonatype repo requiring the same IP. AFAIK, atlas isn't using Travis anymore (but I could definitely be wrong about this), so 9555 isn't an issue. And I don't believe that GitHub actions will rotate IP address per upload.

It _should_ work, but I've never published to Sonatype. I want to say one of the other projects I have worked on did something similar, but I cannot remember which project at this point.